### PR TITLE
Dbot score type file

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/23169

## Description
Change in all of this integrations the dbotscore type hash to have type file as well as you can see in the screenshot.

** Integrations**:
Virustotal
Virustotal-Private-API
Flashpoint
Lastline v2
McAfee Threat Intelligence Exchange
OPSWAT-Metadefender V2
Palo Alto Networks MineMeld
ReversingLabs Titanium Cloud
Symantec Deepsight Intelligence
WildFire-v2

## Screenshots
Screen Shot 2020-04-23 at 16 23 03

## Does it break backward compatibility?
   - [x] No

## Notes
I couldn't check the changes in this integrations: OPSWAT-Metadefender V2, Virustotal-Private-API and WildFire-v2

### CODE REVIEW

File: Beta_Integrations/integration-Symantec_Deepsight.yml [hunk 1]
```diff
@@ -464,12 +464,19 @@ script:
...
-        dbotscore_context = {
+        dbotscore_context = [{
             'Indicator' : filehash,
             'Type' : 'hash',
             'Vendor' : 'Symantec Deepsight Intelligence',
             'Score' : dbotscore
+        },
+        {
+            'Indicator' : filehash,
+            'Type' : 'file',
```
Comment (ronykoz) [inline Beta_Integrations/integration-Symantec_Deepsight.yml:475]:
```suggestion
            'Type' : 'File',
```

File: Packs/Legacy/Integrations/Flashpoint/Flashpoint.py [hunk 2]
```diff
@@ -701,20 +702,38 @@ def file_lookup_command(client, file):
...
+                'Vendor': 'Flashpoint',
+                'Score': 3
+            },
 
-        file_context = {
-            indicator_type: file,
-            'Malicious': {
+            {
+                'Indicator': file,
+                'Type': 'file',
```
Comment (ronykoz) [inline Packs/Legacy/Integrations/Flashpoint/Flashpoint.py:715]:
```suggestion
                'Type': 'File',
```

File: Packs/Legacy/Integrations/Flashpoint/Flashpoint.py [hunk 3]
```diff
@@ -701,20 +702,38 @@ def file_lookup_command(client, file):
...
+        file_context = [
+            {
+                indicator_type: 'hash',
+                'Malicious': {
+                    'Vendor': 'Flashpoint',
+                    'Description': 'Found in malicious indicators dataset'
+                }
+            },
+            {
+                indicator_type: 'file',
```
Comment (ronykoz) [inline Packs/Legacy/Integrations/Flashpoint/Flashpoint.py:730]:
```suggestion
                indicator_type: 'File',
```

File: Packs/Legacy/Integrations/Lastline_v2/Lastline_v2.py [hunk 4]
```diff
@@ -224,15 +227,18 @@ def get_report_context(result: Dict, threshold=None) -> Dict:
...
                 dbotscore['Indicator'] = analysis_subject.get('md5')
-                dbotscore['Type'] = 'hash'
                 data['MD5'] = analysis_subject.get('md5')
                 data['SHA1'] = analysis_subject.get('sha1')
                 data['SHA256'] = analysis_subject.get('sha256')
                 data['Type'] = analysis_subject.get('mime_type')
             dbotscore['Type'] = key
+            if key == 'File':
+                dbotscore_copy = dbotscore.copy()
+                dbotscore_copy['Type'] = 'file'
```
Comment (ronykoz) [inline Packs/Legacy/Integrations/Lastline_v2/Lastline_v2.py:237]:
```suggestion
                dbotscore_copy['Type'] = 'File'
```

File: Packs/Legacy/Integrations/Lastline_v2/Lastline_v2.py [hunk 5]
```diff
@@ -159,7 +159,10 @@ def report_generator(result: Dict, threshold=None):
     score = result['data'].get('score')
     uuid = result['data'].get('task_uuid')
     submission_time = result['data'].get('submission')
-    indicator = context_entry.get('DBotScore', {}).get('Indicator', 'None')
+    if key == 'File':
+        indicator = context_entry.get('DBotScore', [{}])[0].get('Indicator', 'None')
+    else:
+        indicator = context_entry.get('DBotScore', {}).get('Indicator', 'None')
```
Comment (ronykoz) [inline Packs/Legacy/Integrations/Lastline_v2/Lastline_v2.py:165]: why?

File: Packs/Legacy/Integrations/Lastline_v2/Lastline_v2.py [hunk 6]
```diff
@@ -248,6 +254,10 @@ def file_hash(path: str) -> str:
     return file_hasher.hexdigest()
 
 
+def return_error(err_msg, error):
+    raise error
```
Comment (ronykoz) [inline Packs/Legacy/Integrations/Lastline_v2/Lastline_v2.py:258]: why?

File: Packs/Legacy/Integrations/Flashpoint/Flashpoint.yml [hunk 7]
```diff
@@ -1,48 +1,51 @@
+category: Data Enrichment & Threat Intelligence
```
Comment (ronykoz) [inline Packs/Legacy/Integrations/Flashpoint/Flashpoint.yml:1]: Can't understand the changes here, most of them seem irrelevant, please download the file from demisto and upload it again

File: Packs/Legacy/Integrations/Lastline_v2/Lastline_v2.py (hunk 5 above)
Comment (barchen1) [reply Packs/Legacy/Integrations/Lastline_v2/Lastline_v2.py:165]: cause the function get_report_context return a list if the type is File

**Comment by barchen1** [comment]
I changed in these two integration the DbotScore type as well.
https://github.com/demisto/content/pull/6579
https://github.com/demisto/content/pull/6579

**Comment by ronykoz** [comment]
@barchen1 let's fix the conflicts and let me know after that so i will force merge

**Comment by ronykoz** [comment]
Force merging due to build problems
